docs/ref/utils.txt

```diff
@@ -331,6 +331,28 @@ https://web.archive.org/web/20110718035220/http://diveintomark.org/archives/2004
 
     See https://web.archive.org/web/20110514113830/http://diveintomark.org/archives/2004/05/28/howto-atom-id
 
+``Stylesheet``
+--------------
+
+.. versionadded:: 5.1
+
+.. class:: Stylesheet
```

Comment:
I realise this is how it is documented below, but we mostly don't document the `__init__` method this way and do things like:
```suggestion
.. class:: Stylesheet(url, mimetype="", media="screen")
```
And then define each of the arguments like:
```
    .. attribute:: url

        The URL of the stylesheet. This argument is required.
```
I quite like this as you can then link to the specific arguments.
Example: https://docs.djangoproject.com/en/5.0/ref/contrib/postgres/aggregates/#arrayagg

Reply:
You're right, it's much more usable that way. One thing to note (it might not be a huge deal but I thought I'd flag it just in case): while `media` is indeed a true attribute, the other two are `@property` so if someone ever tries to instantiate a `Stylesheet` then change the attributes afterwords then they'll get an error. It should be possible to add setters to make the attributes work transparently, but I wasn't sure if it was worth the added complexity. What do you think?

Reply:
I might be wrong, but I think we do this mostly for formatting rather than to tell the user it is an attribute vs a property 🤔 so I don't think the docs should drive any requirement for setters

Reply:
Excellent, less work for me 😁